virt/logging_events_monitoring/virt-running-cluster-checkups.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{VirtProductName} {VirtVersion} includes a diagnostic framework to run predefined checkups that can be used for cluster maintenance and troubleshooting.
+{VirtProductName} includes a diagnostic framework to run predefined checkups that can be used for cluster maintenance and troubleshooting.
```

Comment:
Because the framework was removed (as a container), I think we should remove references to it.
We currently have one checkup, but more are in the making.

```suggestion
{VirtProductName} includes predefined checkups that can be used for cluster maintenance and troubleshooting.
```